project/Dependencies.scala

```diff
@@ -188,7 +192,7 @@ object Dependencies {
         Provided.activation // dockerClient needs javax.activation.DataSource in JDK 11+
       )
 
-  val actorTestkitTyped = l ++= Seq(Provided.logback, Provided.junit, Provided.scalatest.value)
+  val actorTestkitTyped = l ++= Seq(Provided.logback, Provided.junit, Provided.scalatest.value, Provided.scalatestJUnit.value)
```

Comment:
is scalatestJUnit for `org.scalatestplus.junit.JUnitSuite`?
That is not used in any of the published `src/main/*` so it should be enough to have it scoped to "test" rather than "provided".

Reply:
fixed.

Reply:
Hm.. this weird, I cannot reproduce build fail on my box via `clean`, `test:compile`, `multi-jvm:compile`.

Reply:
stange, it should be ordinary test:compile
```
[info] Compiling 143 Scala sources and 27 Java sources to /home/travis/build/akka/akka/akka-actor-tests/target/scala-2.12/test-classes ...
395[error] /home/travis/build/akka/akka/akka-actor-tests/src/test/java/akka/actor/AddressTest.java:11:1: package org.scalatest.junit does not exist
396[error] org.scalatest.junit.JUnitSuite
397[error] /home/travis/build/akka/akka/akka-actor-tests/src/test/java/akka/actor/AddressTest.java:17:1: cannot find symbol
398[error]   symbol: class JUnitSuite
```

btw, `validateCompile` is what pr-validation is using. I think that is for multi-jvm, but multi-jvm:compile should also be fine